src/commands/buildCommands/server.js

```diff
@@ -65,21 +65,36 @@ function writeServerAppMap(localeCodes, isCombined) {
 	);
 }
 
+const addBabelOption = yargs => {
+	yargs.option('babel', {
+		alias: 'config',
+		description: 'path for babel config',
+		demandOption: true,
+		type: 'string'
+	});
+};
+
 module.exports = {
 	command: 'server',
 	description: 'build the server-side renderer bundle',
-	builder: yargs => addLocalesOption(yargs),
+	builder: yargs => {
+		addLocalesOption(yargs);
+		addBabelOption(yargs);
+	},
 	handler: argv => {
 		console.log(
 			chalk.blue('building server bundle using current vendor bundles')
 		);
+
+		const babel = require(argv.babel);
```

Comment:
just a naming consideration, wondering if it would be better to rename `babel` to `babelConfig` wherever it's used, just to make it a little more clear what it's referring to.

Reply:
Yeah the naming convention is weird and I agree it's confusing. I can update this.